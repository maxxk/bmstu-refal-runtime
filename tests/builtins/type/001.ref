$ENTRY Go
{
	= 	<Prout <Type ()>>
		<Prout <Type 'asdwqeqwe'>>
		<Prout <Type 'TEWsdqwe'>>
		<Prout <Type '97872'>>
		<Prout <Type 910230>>
		<Prout <Type >>
		<Prout <Type ''>>
		<Prout <Type 'ЛРфылврлфоыв'>>
		<Prout <Type 'щзгшгЛРфылврлфоыв'>>
		<Prout <Type UUjwerASdasd>>
		<Prout <Type "Ident">>;
}