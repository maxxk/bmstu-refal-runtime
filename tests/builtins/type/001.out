B0 ()
Llasdwqeqwe
LuTEWsdqwe
D0 97872
N0 910230 
*0 
*0 
PuЛРфылврлфоыв
PlщзгшгЛРфылврлфоыв
WiUUjwerASdasd 
WiIdent 
